src/dmd/tokens.d

```diff
@@ -1113,7 +1113,7 @@ nothrow:
         return toString(value).ptr;
     }
 
-    static const(char)* toChars(ushort value)
+    static const(char)* toChars(ubyte value)
```

Comment:
Does this need updating too in tokens.h?

Reply:
Redefining `TOK` does that.

Reply:
Then you maybe need to update the function signature there too

Reply:
Since the testsuite is passing, I guess this is good to go?

Reply:
@WalterBright then this function signature should accept a `TOK` instead then?

Reply:
Ohhh, this function is dead code. Kill it without mercy!

Reply:
We can merge this and then delete it in a subsequent PR.

Reply:
@RazvanN7 I'm waiting for your subsequent PR, it's been 2 minutes and still isn't here yet. ;-)

Reply:
Well, it took some time to make sure that all the appropriate headers are updated: https://github.com/dlang/dmd/pull/13436